graph/src/data/subgraph/mod.rs

```diff
@@ -467,7 +467,8 @@ pub struct MappingEventHandler {
 
 impl MappingEventHandler {
     pub fn topic0(&self) -> H256 {
-        self.topic0.unwrap_or_else(|| string_to_h256(&self.event))
+        self.topic0
+            .unwrap_or_else(|| string_to_h256(&self.event.replace("indexed ", "")))
```

Comment:
We can use [trim_start_matches](https://doc.rust-lang.org/std/primitive.str.html#method.trim_start_matches) to be a bit more precise. We should also trim whitespace if we don't already.